pl_engine/pl_server/src/main/antlr/PlcLexer.g4

```diff
@@ -45,13 +45,15 @@ TRUNCATE:                     T R U N C A T E   { staticSqlParenMatch++; checkFi
 // other keywords
 AND:                          A N D ;
 AS:                           A S ;
+AUTHID:                       A U T H I D ;
 AUTONOMOUS_TRANSACTION:       A U T O N O M O U S '_' T R A N S A C T I O N ;
 BEGIN:                        B E G I N ;
 BETWEEN:                      B E T W E E N ;
 BIGINT:                       B I G I N T ;
 BOOLEAN:                      B O O L E A N ;
 BY:                           B Y ;
 CASE:                         C A S E ;
+CALLER:                       C A L L E R;
```

Comment:
Oracle syntax only allows AUTHID CURRENT_USER or AUTHID DEFINER.
CALLER and OWNER seems unnecessary.

Reply:
While Oracle does not allow the use of CALLER and OWNER, incorporating support for these keywords would be beneficial for users seeking a better understanding of this feature. This is particularly relevant as the term 'OWNER' is extensively used in various contexts within CUBRID, whereas 'DEFINER' is not.